data-and-functions-1/src/functions/getMostExpensiveProduct.js

```diff
@@ -0,0 +1,17 @@
+import DATA from '../DATA'
+
+const getMostExpensiveProduct = (data) => {
+  if (data == null) {
+    return null
+  }
+  let mostExpensiveProduct = 0
+  DATA.products.forEach((m) => {
+    const currentProduct = data.products[m]
+    if (currentProduct.price > mostExpensiveProduct) {
```

Comment:
Two things need to change here:

1. You should test to see if `mostExpensiveProduct == null`. OR:
2. Compare the prices of the `currentProduct` and the `mostExpensiveProduct`

If either of those are true, enter your `if`